bundles/org.openhab.binding.gree/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="gree"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="airconditioner">
+		<label>@text/thing-type.gree.airconditioner.label</label>
+		<channels>
+			<channel id="power" typeId="system.power" />
+			<channel id="mode" typeId="mode" />
+			<channel id="temperature" typeId="temperature" />
+			<channel id="air" typeId="air" />
+			<channel id="dry" typeId="dry" />
+			<channel id="turbo" typeId="turbo" />
+			<channel id="windspeed" typeId="windspeed" />
+			<channel id="quiet" typeId="quiet" />
+			<channel id="swingUpDown" typeId="swingUpDown" />
+			<channel id="swingLeftRight" typeId="swingLeftRight" />
+			<channel id="powersave" typeId="powersave" />
+			<channel id="light" typeId="light" />
+			<channel id="health" typeId="health" />
+		</channels>
+
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<context>network-address</context>
+			</parameter>
+			<parameter name="broadcastAddress" type="text" required="false">
+				<context>network-address</context>
+			</parameter>
```

Comment:
These parameters should have labels.

Reply:
No, works without, because they are included in the properties files and loaded automatically by the framework

![Bildschirmfoto 2020-06-26 um 14 14 49](https://user-images.githubusercontent.com/12601656/85856026-8070dd80-b7b7-11ea-98de-80ddd4d6247e.png)

Reply:
I think it works without because they are defined in the properties files